services/stackexchange/stackexchange-base.spec.js

```diff
@@ -1,12 +1,27 @@
 import Joi from 'joi'
 import { expect } from 'chai'
 import nock from 'nock'
+import { pathParams } from '../index.js'
 import { cleanUpNockAfterEach, defaultContext } from '../test-helpers.js'
 import { StackExchangeBase } from './stackexchange-base.js'
+import StackExchangeMonthlyQuestions from './stackexchange-monthlyquestions.service.js'
+import StackExchangeReputation from './stackexchange-reputation.service.js'
+import StackExchangeQuestions from './stackexchange-taginfo.service.js'
 
 class DummyStackExchangeService extends StackExchangeBase {
```

Comment:
If we're explicitly testing every concrete implementation of a base class, we can actually just get rid of the dummy implementation. I don't think it really adds any value.

Reply:
Good point, I will remove it after we sort the other comments

Reply:
Fixed with https://github.com/badges/shields/pull/9681/commits/9dd597dcf2ab772077b23e9b9918114c88f5384d